service/collector/chart_config_resource.go

```diff
@@ -37,36 +37,36 @@ var (
 	)
 )
 
-// ChartResourceConfig is this collector's configuration struct.
-type ChartResourceConfig struct {
```

Comment:
We need to introduce `Chart` resource after this PR, so I suggest this renaming.